modules/oe_whitelabel_search/templates/block--oe-whitelabel-search-form.html.twig

```diff
@@ -0,0 +1,34 @@
+{#
+/**
+ * @file
+ * Override theme implementation to display block oe_whitelabel-search-form.
+ *
+ * Available variables:
```

Comment:
as we did in the oebt PR lets remove these as they are there in block.html.twig already